src/Slider.jsx

```diff
@@ -428,12 +428,6 @@ export default React.createClass({
 
   // istanbul ignore next
   handleClick(ev) {
-    // if we're coming off of the end of a slide don't handle the click also
```

Comment:
nah I don't think we can do drag testing without writing a lot of code to simulate drag events. I'd rather leave that to the testing library.